src/transpiler.ts

```diff
@@ -99,6 +99,7 @@ export function transform(ast: AST, options: TranspilationOptions & PrintOptions
 
 function applyPasses(ast: AST, options: TranspilationOptions & PrintOptions): AST {
   const passes: Map<string, typeof ASTMapper> = createPassMap([
+    ['E1', ExpressionSplitter],
```

Comment:
@cicr99 which pass would be best to move this after?

Reply:
I keep looking it still prints errors I'm trying to see why